src/main/java/frc/robot/commands/OuttakeSequence.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.Constants;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.Lift.LiftPosition;
+
+public class OuttakeSequence extends SequentialCommandGroup {
+  private Intake intake;
+  private Lift lift;
+  private LiftPosition scoringPosition;
+
+  public OuttakeSequence(Intake intake, Lift lift, LiftPosition scoringPosition) {
```

Comment:
Instead of `scoringPosition` we should have some fixed outtake position. You'll probably need to add this to the Lift class, `LiftPosition` enum, the positions map, etc. It should be considered above start